### PR TITLE
Add massiveref 2020_09, move cleanup and test scripts to subdir

### PR DESCRIPTION
**Pull request by colindaven**

### CODE REVIEW

File: run_Wochenende_SLURM.sh [hunk 1]
```diff
@@ -62,10 +62,16 @@ cpus=12
 
 
 
+# 2020_09 massive reference
+python3 run_Wochenende.py --metagenome 2020_09_massiveref_human --threads $cpus --aligner bwamem --no_abra        --remove_mismatching 3 --readType SE --debug --force_restart $fastq
```
Comment (sannareddyk) [inline run_Wochenende_SLURM.sh:66]: looks fine, please delete space after --no_abra

File: run_Wochenende.py [hunk 2]
```diff
@@ -1124,6 +1130,31 @@ def markDups(stage_infile):
...
+        "-@",
+        IOthreadsConstant,
+        "--output-fmt",
+        "BAM",
+        stage_infile,
+        stage_outfile,
+    ]
+    runStage(stage, stMarkDupsCmd)
+    rejigFiles(stage, stage_infile, stage_outfile)
+    return stage_outfile
```
Comment (sannareddyk) [inline run_Wochenende.py:1153]: looks fine, to be tested with the new massive reference, 2020_09

File: run_Wochenende.py [hunk 3]
```diff
@@ -1517,7 +1551,7 @@ def main(args, sys_argv):
 
     parser.add_argument(
         "--fastp",
-        help="Use tool fastp instead of fastqc and trimmomatic",
+        help="Use fastp trimmer instead of fastqc and trimmomatic",
```
Comment (sannareddyk) [inline run_Wochenende.py:1554]: why instead of fastqc?

**Review by sannareddyk — COMMENTED** [review]
Hi Colin, I have reviewed code to include massiveref_2020_09. New ref and samtools markdup to be tested

File: run_Wochenende.py (hunk 3 above)
Comment (colindaven) [reply run_Wochenende.py:1554]: fastp includes a graphical fastqc style output and is faster than fastqc. We don't use it much so far though.

File: run_Wochenende_SLURM.sh (hunk 1 above)
Comment (colindaven) [reply run_Wochenende_SLURM.sh:66]: The space was added on purpose to make clear there is no --mq30 filter .

**Comment by colindaven** [comment]
Thanks Keerthi. There are a few conflicts but I'll do my best to merge

**Pull request merged**